### DISCUSSION TITLE
Language and Units in Grafana

### DISCUSSION THREAD

**Discussion by samywee** [bassmaster187/TeslaLogger#948] [General]
I have started using TeslaLogger in a docker image using docker-compose as described in the https://github.com/bassmaster187/TeslaLogger/blob/master/docker_setup.md. I am not finding any easy way to change language to English and units to miles when data is displayed in Grafana (Sorry I am in US). Do I have to follow any steps when I install the docker image to change them in Grafana?  Grafana Language / Unit support is available, or it is only for admin panel? 

Appreciate if anyone can help.

**Comment by bassmaster187** [comment]
Go to admin panel and change your language to english and units to mile / fahrenheit.
Wait a minute. And go back to Grafana. Maybe you need to refresh the dashboard. On Windows + Chrome Strg + F5.

![image](https://user-images.githubusercontent.com/6816385/220549548-1dea5520-d44f-4e66-a086-a1ecc15d44a5.png)

If everythings works fine, you should see everything as expected:

![image](https://user-images.githubusercontent.com/6816385/220549991-79de9aca-0bde-4e8e-b95d-3c4edb831765.png)

**Reply by samywee** [reply]
@bassmaster187  Thank you. Interesting..  I see that it is working for some filed values in the "Trip" dashboard / view that you have posted. Here is "Consumption statistics, where it do not seems to change. See the non-English words , Celsius and km still in this report. My settings in the Admin panel are as you have stated. 

Anything you can think of? If I wipe off the docker install and reinstall including docker-volumes will this get fixed? Or is this a bug on the code, that need to be still addressed? Let me know. 

![image](https://user-images.githubusercontent.com/16374316/220761380-497d2158-e7b0-49e1-ac78-78b17a74c764.png)

Here is another issue on the Trip dashboard where Header changed to F but still values did not change from C to F. 
![image](https://user-images.githubusercontent.com/16374316/220786342-e9427062-9ba8-4064-9074-6d733811f332.png)

**Reply by kinteseay** [reply]
Did you ever get this figured out?  I'm having the same issue
